lib/realtime/tenants.ex

```diff
@@ -210,6 +212,23 @@ defmodule Realtime.Tenants do
     {:channel, :presence_events, tenant.external_id}
   end
 
+  @doc """
+  The GenCounter key to use when counting connection attempts against Realtime.Tenants.Connect
+  ## Examples
+    iex> Realtime.Tenants.connection_attempts_per_second_key("tenant_id")
+    {:channel, :connection_attempts, "tenant_id"}
+    iex> Realtime.Tenants.connection_attempts_per_second_key(%Realtime.Api.Tenant{external_id: "tenant_id"})
+    {:channel, :connection_attempts, "tenant_id"}
+  """
+  @spec connection_attempts_per_second_key(Tenant.t() | String.t()) :: {:channel, :connection_attempts, String.t()}
+  def connection_attempts_per_second_key(tenant) when is_binary(tenant) do
+    {:channel, :connection_attempts, tenant}
```

Comment:
Why `:channel` is used here? Is it related to a specific channel in this context?

Reply:
moving it to `{:connection_attempts, tenant}`